secrets-provider/templates/secrets-provider.yaml

```diff
@@ -75,5 +75,15 @@ spec:
         - name: DEBUG
           value: "true"
         {{- end }}
+
+        {{- if .Values.environment.syncIntervalTime }}
+        - name: SYNC_INTERVAL_TIME
+          value: {{ .Values.environment.syncIntervalTime }}
```

Comment:
@oburstein-hub @eladkug what would be the mainstream flow for using SyncIntervalTime and RetryCount? 
1️⃣ What should the details be in code if the user does not provide us with this?
2️⃣ What if the user only provides `syncIntervalTime` and not `retryCount`? Are they a package deal or would the normal flow be taking the configured syncIntervalTime and use the default retryCount if not provided also?

Reply:
1. The defaults should be as follows: SyncIntervalTime: 3 seconds, retryCount: 3 - @InbalZilberman - please comment
2. User can provide each of above parameter separately. The unprovided parameter will stay with default value

Reply:
to clarify, these params are for our retry mechanism if there was a failure (connecting to follower, fetching conjur secret, updating k8s secret, etc)

Reply:
@oburstein-hub  great thank you! Want to note that these default will **not** be forward facing rather be in the code itself